tfx/components/transform/executor.py

```diff
@@ -279,11 +279,15 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           should contain two splits 'train' and 'eval'.
         - schema: A list of type `standard_artifacts.Schema` which should
           contain a single schema artifact.
+        - cache_output_path: Optinal cache input of 'tf.Transform', where cached
```

Comment:
Would it be possible to resolve this TODO in this change?
https://github.com/tensorflow/tfx/blob/deb618730dc7675c8e9dc75e03b8ca795d49653d/tfx/components/transform/executor.py#L124-L129

This is likely needed in order to enable cache in OSS

Reply:
DatsetKey now uses a SHA-256 hash of the file_pattern to generate.